### PR TITLE
Update Django, use pypi releases, add coveralls

### PR DESCRIPTION
**Pull request by jpmckinney**
Also: Not all tests were being run. I had to fix a test that seems to have been copied from BODS; I'm not sure what it's supposed to be testing, but at least it passes now.

### CODE REVIEW

**Comment by jpmckinney** [comment]
Looks like .travis.yml always installs the most recent version of Chrome, which means we'll regularly have to update the Chrome driver. We should instead always install the same version of Chrome. Related: #35

**Comment by jpmckinney** [comment]
@michaelwood I maybe misunderstand what the footer test is meant to actually test, but anyhow, it's failing. Can you have a look?

**Comment by jpmckinney** [comment]
It looks like the footer test was introduced in 2015, and was acknowledged as maybe a bad idea in the long term: https://github.com/OpenDataServices/cove/commit/a4631ce18e427d8ca43b48aa00c6b77ed89ee47b

I'd agree that testing for text on a site outside the control of the present repository is a brittle test. I also don't think we need to test that clicking links work (we similarly don't need to test whether the browser's back button works, etc.).

File: .travis.yml [hunk 1]
```diff
@@ -11,27 +11,30 @@ cache: pip
...
   - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16"
   - sleep 3 # give xvfb some time to start
-  - set -e # make sure we exit if one command fails
 
 script:
+  - curl -s -S --retry 3 $BASEDIR/tests/script.sh | bash -
   - "google-chrome --version"
   - "google-chrome &"
-  - "DJANGO_SETTINGS_MODULE=cove_project.settings  python3 -m pytest ./tests/"
-  - "flake8"
```
Comment (robredpath) [inline .travis.yml]: @jpmckinney what's the rationale for removing flake8 from the automated test? On our stuff, we've found it a helpful way of catching anything that slips through - eg forgetting to run flake8 after fixing other test fails.

Comment (jpmckinney) [reply .travis.yml]: It's contained in `curl -s -S --retry 3 $BASEDIR/tests/script.sh | bash -` which is run on all other OCP repositories.

**Comment by michaelwood** [comment]
@jpmckinney  Sorry I have to decline the review request at the moment as I'm currently not scheduled to be working on this. I believe there is a trello? board for dev time requests where another developer could pick this up as a piece of work. Thanks

**Comment by jpmckinney** [comment]
Thanks for the note, @michaelwood. It's attached to a card in Trello, so I'll wait for a reviewer.

**Pull request merged**